var/spack/repos/builtin/packages/gimp/package.py

```diff
@@ -71,6 +72,7 @@ class Gimp(AutotoolsPackage):
     depends_on("libexif")
     # depends_on("libheif+libde265", when="+libheif")
     depends_on("libjxl", when="+jpegxl")
+    depends_on("libjxl@:0.7", when="+jpegxl@:2.10.32")
```

Comment:
@wdconinc JFYI, Instead of this carrying this version constraint, I guess we could remove the older gimp versions (`gimp-2.10.32` and before) from the recipe if we have no need for them. (or make them deprecated?)